internal/provisioner/plain/controllers/bundledeployment_controller.go

```diff
@@ -132,7 +132,12 @@ func (r *BundleDeploymentReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		if bundle.Status.Phase == rukpakv1alpha1.PhaseFailing {
 			reason = rukpakv1alpha1.ReasonUnpackFailed
 			status = metav1.ConditionFalse
-			message = fmt.Sprintf("Failed to unpack the %s Bundle", bundle.GetName())
+			c := meta.FindStatusCondition(bundle.Status.Conditions, rukpakv1alpha1.TypeUnpacked)
+			bundleMessage := "No unpacked condition"
+			if c != nil {
+				bundleMessage = c.Message
+			}
+			message = fmt.Sprintf("Failed to unpack the Bundle: %s, reason: %s", bundle.GetName(), bundleMessage)
```

Comment:
nit: I think we can drop the ", reason: ..." here and just refactor the message to `fmt.Sprintf("Failed to unpack the %s Bundle: %s", bundle.GetName(), bundleMessage)`. WDYT?

Reply:
It's good.  I'll make change.  Thanks!